tests/test_parseCalibsGen2.py

```diff
@@ -0,0 +1,167 @@
+        self.flatFile = os.path.join(testDataDirectory, "pipeline-calibs", "FLAT", "FLAT-g-47.fits")
+
+        # DECam Community Pipeline calibration products, also full of zeros
+        cpBiasFilename = "c4d_150218_191721_zci_v1.fits.fz"
+        self.cpBiasFile = os.path.join(testDataDirectory, "hits2015-zeroed", cpBiasFilename)
+        cpFlatFilename = "c4d_150218_200522_fci_g_v1.fits.fz"
+        self.cpFlatFile = os.path.join(testDataDirectory, "hits2015-zeroed", cpFlatFilename)
+
+        # A Butler repository with ingested raws and Community Pipeline calibs
+        repo = os.path.join(testDataDirectory, "ingested")
+        calibRepo = os.path.join(testDataDirectory, "calibingested")
+        self.butler = dafPersist.Butler(root=repo, calibRoot=calibRepo)
+
+    def testLsstCalibsGetInfo(self):
+        """Test that the filepath and a calib_hdu value of 1 is in the info.
+        """
+        # LSST-Science-Pipelines-built bias
+        phuInfo, infoList = self.task.getInfo(self.biasFile)
+        assert phuInfo['calib_hdu'] == 1
+        assert 'path' in phuInfo
+        for item in infoList:
+            assert item['calib_hdu'] == 1
+            assert 'path' in item
+
+        # LSST-Science-Pipelines-built flat
+        phuInfo, infoList = self.task.getInfo(self.flatFile)
+        assert phuInfo['calib_hdu'] == 1
+        assert 'path' in phuInfo
+        for item in infoList:
+            assert item['calib_hdu'] == 1
+            assert 'path' in item
+
+    def testLsstCalibsGetDestination(self):
+        """Test that file path destinations conform to the mapper templates.
+        """
+        # LSST-Science-Pipelines-built bias
+        dataId = {'ccdnum': 1, 'date': '2015-02-18'}
+        filename = self.task.getDestination(self.butler, dataId, self.biasFile)
+        self.assertIn(f"BIAS/{dataId['date']}/BIAS-{dataId['date']}-{dataId['ccdnum']:02}.fits", filename)
+        self.assertNotIn('[', filename)
+
+        # LSST-Science-Pipelines-built flat
+        dataId = {'ccdnum': 1, 'date': '2015-02-18', 'filter': 'g'}
+        filename = self.task.getDestination(self.butler, dataId, self.flatFile)
+        self.assertIn(f"FLAT/{dataId['date']}/{dataId['filter']}/"
+                      + f"FLAT-{dataId['date']}-{dataId['ccdnum']:02}.fits", filename)
+        self.assertNotIn('[', filename)
+
+    def testLsstCalibsTranslateFilter(self):
+        """Test that filter is set to "NONE" for the bias and "g" for the flat.
+        """
+        # LSST-Science-Pipelines-built bias
+        md = readMetadata(self.biasFile, self.task.config.hdu)
+        biasFilter = self.task.translate_filter(md)
+        assert biasFilter == "NONE"
+
+        # LSST-Science-Pipelines-built flat
+        md = readMetadata(self.flatFile, self.task.config.hdu)
+        flatFilter = self.task.translate_filter(md)
+        assert flatFilter == "g"
+
+    def testCpCalibsGetInfo(self):
```

Comment:
Ugh, I really don't like this camelCasing, but I guess it's technically correct per our dev guide. grumble_underscores_are_better_grumble...